src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -282,27 +283,67 @@ specified using a single shape topology for each element shape.
+To conform to protocol, each entry in the ``matsets`` section must be an *Object* that contains the following information:
+
+   * matsets/matset/topology: "topo"
+   * matsets/matset/volume_fractions: (mcarray)
+
+
+
 Fields
 ++++++++++++++++++++
 
-Fields are used to hold simulation state arrays associated with a mesh topology.
+Fields are used to hold simulation state arrays associated with a mesh topology and (optionally) a mesh material set.
 
+Each field entry can define an **mcarray** of material-independent values and/or an **mcarray** of per-material values.
+These data arrays must be specified alongside a source space, which specifies the space over which the field values are defined (i.e. a topology for material-independent values and a material set for material-dependent values).
+Minimally, each field entry must specify one of these data sets, the source space for the data set, an association type (e.g. per-vertex, per-element), and a volume scaling type (e.g. volume-dependent, volume-independent).
+Thus, to conform to protocol, each entry under the ``fields`` section must be an *Object* that adheres to one of the following descriptions 
 
-A field contains an **mcarray** and information about how this data is associated with elements of the topology. 
-To conform to the protocol, each entry under ``fields`` must be an *Object* that contains one of these two styles of field descriptions:
+ * Material-Independent Fields:
 
- * Standard Fields:
+   * fields/field/association: "vertex" | "element" | (mfem-style finite element collection name)
+   * fields/field/volume_dependent: "true" | "false"
+   * fields/field/topology: "topo"
+   * fields/field/values: (mcarray)
 
-   * fields/den/topology: "topo" 
-   * fields/den/association: “vertex” | "element"
-   * fields/den/values: (mcarray)
+ * Material-Dependent Fields:
+
+   * fields/field/association: "vertex" | "element" | (mfem-style finite element collection name)
+   * fields/field/volume_dependent: "true" | "false"
+   * fields/field/matset: "matset"
+   * fields/field/matset_values: (mcarray)
+
+ * Mixed Fields:
+
+   * fields/field/association: "vertex" | "element" | (mfem-style finite element collection name)
+   * fields/field/volume_dependent: "true" | "false"
+   * fields/field/topology: "topo"
+   * fields/field/values: (mcarray)
+   * fields/field/matset: "matset"
+   * fields/field/matset_values: (mcarray)
+
+
+
+Adjacency Sets
+++++++++++++++++++++
 
+Adjacency Sets are used to outline the shared geometry between subsets of domains in multi-domain meshes.
 
- * High Order Fields:
+Each entry in the Adjacency Sets section represents an adjacency group, which is meant to encapsulate a set of adjacency information shared between domains.
+Each individual adjacency group contains a source topology, an element association, and list of shared element groups (which themselves contain a subset of adjacent neighbor domains IDs and a list of shared element IDs).
```

Comment:
the example is correct, but I think this text needs to be udpated. 
It says each adj group specifies a topo, now its each adj set (prior bullet)